modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -32,13 +36,18 @@ def type
           active_itf = intent_to_files.detect do |itf|
             itf[:status].casecmp?('active') && itf[:expiration_date].to_datetime > Time.zone.now
           end
-
+          ClaimsApi::Logger.log('ITF', type: params[:type].downcase,
+                                       detail: 'ift resource not found')
```

Comment:
Similar to above, getting "itf resource not found" in cases where one is found.

Using: http://localhost:3000/services/claims/v2/veterans/1012667145V762142/intent-to-file/:type

Logged: 2023-08-18 09:33:00.908961 I [10318:puma srv tp 008] Rails -- ClaimsApi :: ITF :: {"type":"survivor","detail":"ift resource not found"} :: Location: /Users/matthewchristianson/development/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb:39

Reply:
Spelling here also: detail: 'ift resource not found', should be "itf"